shared/modules/persistence_module.go

```diff
@@ -23,6 +24,10 @@ type PersistenceModule interface {
 
 	// BlockStore operations
 	GetBlockStore() blockstore.BlockStore
+
+	// TreeStore operations
```

Comment:
I know we didn't do a good job at this earlier on, but can make the comments more explicit on what it does and a couple words on what it's used for. It'll help future readers

Reply:
Update incoming  👍